### PR TITLE
feat(OnyxSelect): Implement description for OnyxSelect

### PR DESCRIPTION
**Pull request by MajaZarkova**
Relates to #1392 

Added new 'listDescription' property. The description text will be shown at the end of the flyout

## Checklist

- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

File: packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue [hunk 1]
```diff
@@ -380,6 +380,13 @@ const selectInputProps = computed(() => {
           <slot name="optionsEnd"></slot>
         </div>
       </div>
+      <div
+        v-if="props.listDescription"
+        :aria-description="props.listDescription"
```
Comment (larsrickert) [inline packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue:385]: Shouldn't the aria-description rather be set on the list/flyout itself?

**Review by larsrickert — CHANGES_REQUESTED** [review]

File: packages/sit-onyx/playwright/snapshots/components/OnyxSelect/Select-list-description--chromium-linux.png
Comment (larsrickert) [inline packages/sit-onyx/playwright/snapshots/components/OnyxSelect/Select-list-description--chromium-linux.png:1]: The vertical spacing of the description looks a little tight to me, can you double check this with @jannick-ux ?

File: packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue (hunk 1 above)
Comment (MajaZarkova) [reply packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue]: @larsrickert I integrated the attribute in the headless composable.

**Review by larsrickert — APPROVED** [review]

File: packages/sit-onyx/playwright/snapshots/components/OnyxSelect/Select-list-description--chromium-linux.png
Comment (MajaZarkova) [reply packages/sit-onyx/playwright/snapshots/components/OnyxSelect/Select-list-description--chromium-linux.png:1]: @larsrickert Martin's decision was to keep the white space, and I will create a follow up ticket for Jannick so he can think about the open questions we have and create a workaround in Figma aligned with his decision.

File: packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue [hunk 2]
```diff
@@ -489,6 +493,16 @@ const selectInputProps = computed(() => {
       color: var(--onyx-color-text-icons-primary-intense);
     }
 
+    &__description {
+      display: flex;
+      padding: var(--onyx-density-3xs) var(--onyx-density-sm);
+      justify-content: flex-end;
+      text-align: right;
+      align-items: center;
+      gap: var(--onyx-spacing-md, 16px);
```
Comment (larsrickert) [inline packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue:502]:
```suggestion
      gap: var(--onyx-spacing-md);
```

**Review by larsrickert — CHANGES_REQUESTED** [review]

**Review by larsrickert — APPROVED** [review]

**Pull request merged**